README.md

```diff
@@ -736,6 +742,154 @@ No errors detected
+
+### Catena Script File
+
+A number of provisioning scripts are provided for setting up Catenas; the
+files are named as `{script}.cat`. If your Catena has already been set up at
+the factory, you can use `catena-otaa.cat`.  Here is the base script used for
+sigfox network:
+
+* `catena-sigfox-base-otaa.cat` - Configure a Catena for OTAA
+
+The scripts conventionally get information from variables that are set up
+by you or by the script. The variables are:
+
+* `BASENAME` - The base name to be used for devices. This must be a legal
+DNS-like name (letters, digits and dashes). The device EUI is appended to
+the name. If you want a dash as a separator between the basename and the
+DEVEUI, you must end the basename value with a dash.
+* `DEVTYPEID` - The device type id is a unique identifier for the application on the network.
+* `PAC` - The PAC is a unique value for proves the ownership of end-device on the network.
+* `KEY` - The key is a unique identifier which used to secure the communication between the device and the network. It is only known by the device and by the application.
+
+### Example (`mcci-catena-provision-sigfox.py`)
+
+```console
+python mcci-catena-provision-sigfox.py -D -port COM32 -permissive -r -V BASENAME=device- -V DEVID=26240FE -V KEY=0123456789ABCDEF0123456789ABCDEF -V PAC=FEDCBA0987654321 -V DEVTYPEID=501f6a9a4175811910adf528 -s catena-sigfox-base-otaa.cat
+
+Port COM32 opened
+>>> system echo off
+
+<<< system echo off
+OK
+
+CheckComms
+>>> system version
+
+Command sent: system version
+
+<<< Board: Catena 4612
+Platform-Version: 0.19.0.30
+Arduino-LoRaWAN-Version: 0.8.0
+Arduino-LMIC-Version: 3.3.0
+MCCIADK-Version: 0.2.2
+MCCI-Arduino-BSP-Version: 2.8.0
+OK
+
+Board: Catena 4612
+Platform-Version: 0.19.0.30
+Arduino-LoRaWAN-Version: 0.8.0
+Arduino-LMIC-Version: 3.3.0
+MCCIADK-Version: 0.2.2
+MCCI-Arduino-BSP-Version: 2.8.0
+
+
+>>> system configure syseui
+
+<<< 00-02-cc-01-00-00-06-22
+OK
+
+
+ Catena Type: Catena 4612
+ Platform Version: 0.19.0.30
+ SysEUI: 0002cc0100000622
+
+Device Created:
+{'id': '26240FE'}
+
+Device Info:
+{'id': '26240FE', 'name': 'device-26240FE', 'satelliteCapable': False, 'repeater
+': False, 'messageModulo': 4096, 'state': 0, 'comState': 5, 'pac': 'FEDCBA098765
+4322', 'location': {'lat': 0.0, 'lng': 0.0}, 'deviceType': {'id': '501f6a9a41758
+11910adf528'}, 'group': {'id': '4ab30f462564325177713118'}, 'lqi': 4, 'creationT
+ime': 1617197032746, 'modemCertificate': {'id': '5fabcd5d80bd5658641e4415'}, 'pr
+ototype': True, 'automaticRenewal': True, 'automaticRenewalStatus': 1, 'createdB
+y': '5eb5a674e833d96c28dffafd', 'lastEditionTime': 1617197032503, 'lastEditedBy'
+: '5eb5a674e833d96c28dffafd', 'activable': True}
+
+DoScript: catena-sigfox-base-otaa.cat
+
+>>> sigfox configure devid 26240FE
+
+<<< OK
+
+>>> sigfox configure pac FEDCBA0987654322
+
+<<< OK
+
+>>> sigfox configure key 0123456789ABCDEF0123456789ABCDEF

```

Comment:
The program should not print the key, since it's a critical secret. The logger should be enhanced to display `**REDACTED**` or otherwise prevent logging of `sigfox configure key ...` commands. 

If the key has to be recorded, it needs to be recorded in a separate file.